scripts/pre_commit_linter.py

```diff
@@ -1927,6 +1933,99 @@ def _check_for_copyright_notice(self):
 
         return summary_messages
 
+    def _check_codeowner_file(self):
+        """Checks the CODEOWNERS file for any uncovered dirs/files and also
+        checks that every pattern in the CODEOWNERS file matches at least one
+        file/dir. Note that this checks the CODEOWNERS file according to the
+        glob patterns supported by Python2.7 environment. For more information
+        please refer https://docs.python.org/2/library/glob.html.
+        """
+        if self.verbose_mode_enabled:
+            print 'Starting CODEOWNERS file check'
+            print '----------------------------------------'
+
+        with _redirect_stdout(_TARGET_STDOUT):
+            codeowner_file = '.github/CODEOWNERS'
+            failed = False
+            summary_messages = []
+            # Checks whether every pattern in the CODEOWNERS file matches at
+            # least one dir/file.
+            path_patterns = []
+            for line_num, line in enumerate(FileCache.readlines(
+                    codeowner_file)):
+                if line.strip() != '' and line.strip()[0] != '#':
```

Comment:
Pull `line.strip()` out in a variable and use it for both the sub-conditions.

Reply:
Done.